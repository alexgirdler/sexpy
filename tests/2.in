(* 1 3 3 7 42 24 42)
(- 59 10  69  7 100 41  15  34  71  27  34  100 82  8 44  50  98  38  47  9 50  35  28  66  32  31  11  2 1 30  72  85  28  4 31  21  20  14  97  28  30  47  78  21  61  31  61  85  1 64  47  63  62  16  37  89  91  27  66  93  94  35  60  82  28  71  51  4 45  91  46  81  74  2 21  82  69  46  8 41  63  61  40  22  36  82  3 52  74  13  76  19  8 97  2 27  53  39  92  79)
(+ 99 29  77  79  87  50  66  46  75  89  80  26  9 66  50  78  99  27  91  46  1 2 55  74  27  68  7 43  39  49  46  95  36  31  25  28  64  59  53  21  34  31  50  78  90  2 63  38  74  61  57  75  61  74  34  72  31  28  4 68  92  38  13  44  58  9 55  20  92  74  3 95  66  81  80  20  24  15  79  61  62  98  83  78  97  32  10  7 69  66  32  23  23  6 27  71  100 6 35  53)
(/ 42 3)
