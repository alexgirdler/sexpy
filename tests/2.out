2667168.0
-4668
5114
14.0
